### PR TITLE
Fixed summary links

### PR DESCRIPTION
**Pull request by salvioner**
fixed broken internal links (minor fix) in articles/connections/database/custom-db/templates/verify.md

### CODE REVIEW

**Review by mpaktiti — APPROVED** [review]

File: articles/connections/database/custom-db/templates/verify.md [hunk 1]
```diff
@@ -26,8 +26,8 @@ When working on your user creation script, keep in mind that:
 Auth0 provides sample scripts for use with the following languages/technologies:
 
 * [JavaScript](#javascript)
-* [ASP.NET Membership Provider (MVC3 - Universal Providers)](#aspnet-membership-provider-mvc3---universal-providers)
-* [ASP.NET Membership Provider (MVC4 - Simple Membership)](#aspnet-membership-provider-mvc4---simple-membership)
+* [ASP.NET Membership Provider (MVC3 - Universal Providers)](#asp-net-membership-provider-mvc3-universal-providers-)
```
Comment (aarighi) [inline articles/connections/database/custom-db/templates/verify.md]: @mpaktiti seems that those new links are broken again, is there a reason for that?

Comment (mpaktiti) [reply articles/connections/database/custom-db/templates/verify.md]: Hey @aarighi They work for me. Maybe the fix wasn't deployed on the live environment when you checked. Check again?
https://auth0.com/docs/connections/database/custom-db/templates/verify#asp-net-membership-provider-mvc3-universal-providers-
https://auth0.com/docs/connections/database/custom-db/templates/verify#asp-net-membership-provider-mvc4-simple-membership-

Thanks!

Comment (aarighi) [reply articles/connections/database/custom-db/templates/verify.md]: Sorry, both links work (: I was checking only the .md file... In the markdown the links don't work, but as long as the website is working fine that's a minor issue, I guess.

Comment (mpaktiti) [reply articles/connections/database/custom-db/templates/verify.md]: No worries :) Thanks for helping in fixing this! 🙌

**Pull request merged**